<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnschlussmatrixWindow</class>
 <widget class="QMainWindow" name="AnschlussmatrixWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="settings_page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="allgemein_group">
            <property name="title">
             <string>Allgemein</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="bahnhofLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>&amp;Bahnhof</string>
               </property>
               <property name="buddy">
                <cstring>bahnhofBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="bahnhofBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="umsteigezeitLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>&amp;Umsteigezeit</string>
               </property>
               <property name="buddy">
                <cstring>umsteigezeitSpin</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="umsteigezeitSpin">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="suffix">
                <string> Min.</string>
               </property>
               <property name="value">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="anschlusszeitLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>&amp;Anschlusszeit</string>
               </property>
               <property name="buddy">
                <cstring>anschlusszeitSpin</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="anschlusszeitSpin">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="suffix">
                <string> Min.</string>
               </property>
               <property name="value">
                <number>15</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="allgemein_spacer_widget" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="ankunft_group">
            <property name="title">
             <string>Ankommende Züge</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="ankunft_filter_label">
               <property name="text">
                <string>Kategorie</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="ankunft_filter_view">
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="ankunft_filter_label_2">
               <property name="text">
                <string>Von</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="von_filter_view">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="abfahrt_group">
            <property name="title">
             <string>Abfahrende Züge</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="abfahrt_filter_label">
               <property name="text">
                <string>Kategorie</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="abfahrt_filter_view">
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="abfahrt_beschriftung_label">
               <property name="text">
                <string>Nach</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="nach_filter_view">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="display_page">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="displaySplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="grafikWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QLabel" name="zuginfoLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Zuginfo: (keine Auswahl)</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSetup"/>
   <addaction name="actionAnzeige"/>
   <addaction name="separator"/>
   <addaction name="actionZugAusblenden"/>
   <addaction name="actionZugEinblenden"/>
   <addaction name="separator"/>
   <addaction name="actionPlusEins"/>
   <addaction name="actionMinusEins"/>
   <addaction name="actionFix"/>
   <addaction name="separator"/>
   <addaction name="actionAnkunftAbwarten"/>
   <addaction name="actionAbfahrtAbwarten"/>
   <addaction name="actionAnschlussAufgeben"/>
   <addaction name="actionLoeschen"/>
  </widget>
  <action name="actionSetup">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/equalizer.png</normaloff>
     <disabledoff>:/equalizer-dis.png</disabledoff>:/equalizer.png</iconset>
   </property>
   <property name="text">
    <string>Einstellungen</string>
   </property>
   <property name="toolTip">
    <string>Bahnhofauswahl (S)</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionAnzeige">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/slots.png</normaloff>
     <disabledoff>:/slots-dis.png</disabledoff>:/slots.png</iconset>
   </property>
   <property name="text">
    <string>Grafik</string>
   </property>
   <property name="toolTip">
    <string>Grafik anzeigen (G)</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionPlusEins">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/clock--plus.png</normaloff>:/clock--plus.png</iconset>
   </property>
   <property name="text">
    <string>+1</string>
   </property>
   <property name="toolTip">
    <string>Verspätung/Wartezeit um 1 Minute erhöhen (+)</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionMinusEins">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/clock--minus.png</normaloff>:/clock--minus.png</iconset>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
   <property name="toolTip">
    <string>Verspätung/Wartezeit um 1 Minute reduzieren (-)</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionFix">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/clock--pencil.png</normaloff>:/clock--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Fix</string>
   </property>
   <property name="toolTip">
    <string>Feste Verspätung festlegen (V)</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionLoeschen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/chain--return.png</normaloff>:/chain--return.png</iconset>
   </property>
   <property name="text">
    <string>Verknüpfung auflösen</string>
   </property>
   <property name="toolTip">
    <string>Verknüpfung auf ausgewählten Anschlüssen auflösen (Del)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAnkunftAbwarten">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/chain--arrow-in.png</normaloff>:/chain--arrow-in.png</iconset>
   </property>
   <property name="text">
    <string>Ankunft abwarten</string>
   </property>
   <property name="toolTip">
    <string>Ankunft von Zubringer abwarten (K)</string>
   </property>
   <property name="shortcut">
    <string>K</string>
   </property>
  </action>
  <action name="actionAbfahrtAbwarten">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/chain--arrow.png</normaloff>:/chain--arrow.png</iconset>
   </property>
   <property name="text">
    <string>Abfahrt abwarten</string>
   </property>
   <property name="toolTip">
    <string>Abfahrt von Zubringer abwarten (F)</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionWarnungSetzen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/flag.png</normaloff>:/flag.png</iconset>
   </property>
   <property name="text">
    <string>Warnung</string>
   </property>
   <property name="toolTip">
    <string>Warnung setzen (W)</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionWarnungIgnorieren">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/flag-green.png</normaloff>:/flag-green.png</iconset>
   </property>
   <property name="text">
    <string>Ignorieren</string>
   </property>
   <property name="toolTip">
    <string>Warnung ignorieren (I)</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionWarnungReset">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/flag-white.png</normaloff>:/flag-white.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Warnung zurücksetzen (R)</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionAnschlussAufgeben">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/chain--cross.png</normaloff>:/chain--cross.png</iconset>
   </property>
   <property name="text">
    <string>Anschluss aufgeben</string>
   </property>
   <property name="toolTip">
    <string>Anschluss nicht abwarten (A)</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionZugAusblenden">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/train--minus.png</normaloff>:/train--minus.png</iconset>
   </property>
   <property name="text">
    <string>Zug ausblenden</string>
   </property>
   <property name="toolTip">
    <string>Zug ausblenden (H)</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionZugEinblenden">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/train--return.png</normaloff>:/train--return.png</iconset>
   </property>
   <property name="text">
    <string>Zug einblenden</string>
   </property>
   <property name="toolTip">
    <string>Alle Züge einblenden (Z)</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
